src/neo/Ledger/ContractState.cs

```diff
@@ -14,47 +14,43 @@ public class ContractState : ICloneable<ContractState>, ISerializable, IInterope
     {
         public int Id;
         public byte[] Script;
+        public ContractAbi Abi;
```

Comment:
Will it be better that replace `Script` and `ContractAbi` by `NefFile`?

Reply:
we will spend more time in deserialize it and we won't need this information inside the chain, I think